File: mambo-shared/mambo-protocol/mambo-protocol-client/src/main/java/org/mambo/protocol/client/messages/MapRunningFightListRequestMessage.java


// Generated on 01/04/2013 14:54:25
package org.mambo.protocol.client.messages;

import java.util.*;
import org.mambo.protocol.client.types.*;
import org.mambo.protocol.client.enums.*;
import org.mambo.protocol.client.*;
import org.mambo.core.io.*;

public class MapRunningFightListRequestMessage extends NetworkMessage {
    public static final int MESSAGE_ID = 5742;
    
    @Override
    public int getNetworkMessageId() {
        return MESSAGE_ID;
    }
    
    
    public MapRunningFightListRequestMessage() { }
    
    
    @Override
    public void serialize(DataWriterInterface writer) {
    }
    
    @Override
    public void deserialize(DataReaderInterface reader) {
    }
    
}

File: mambo-protocol/src/main/scala/org/mambo/protocol/messages/game/context/roleplay/MapRunningFightListRequestMessage.scala





















// Generated on 05/03/2013 10:34:42
// Scala version by Blackrush
package org.mambo.protocol.messages

import org.mambo.core.io._
import org.mambo.protocol.{Message => BaseMessage, _}
import org.mambo.protocol.types._
import org.mambo.protocol.enums._

trait MapRunningFightListRequestMessage extends MessageDeserializer {
  val messageId = 5742

  protected trait Message extends BaseMessage with Serializable {
    val messageId = MapRunningFightListRequestMessage.messageId


    def serialize(buf: Buffer) {
    }

  }

  protected def create(): Target = throw new IllegalAccessException

  def deserialize(buf: Buffer) = {

    create()
  }

}

object MapRunningFightListRequestMessage extends MapRunningFightListRequestMessage {
  case class MessageImpl() extends Message
  type Target = MessageImpl

  override def create() = MessageImpl()

  def apply() = create()
  def unapply(msg: MessageImpl) = MessageImpl.unapply(msg)
}

